rest/specs/src/main/filtered-resources/openapi/openapi.yaml

```diff
@@ -202,6 +383,14 @@ components:
           minLength: 1
           maxLength: 2048
           example: https://www.example.com/mis/30f14c6c1fc85cba12bfd093aa8f90e3
+        finalLocation:
+          description: >
+            This is a URI that will only be used by the Transform service to put on the transformed
```

Comment:
✏️ `put in`

Reply:
Do we need to say here how we will use it? I am thinking we should say that this should be the permanent URI (as opposed to temporary above) corresponding to the resource from which it can later be retrieved.